### DISCUSSION TITLE
football_players_pose_estimation- 02. Utils

### DISCUSSION THREAD

**Discussion by kenany66** [SkalskiP/sports#3] [General]
I have an error in football players pose estimation - 02. Utils, how do I fix this error? You can see this error in this uploaded image.
https://github.com/kenany66/code-gymnastics/blob/main/pose.png

**Comment by SkalskiP** [comment]
Hi, @kenany66 👋🏻! That exception is usually raised when there are no more elements to be generated. Interestingly if I understand correctly, it happens to you right away. Are you sure `SOURCE_VIDEO_A_PATH` is a valid path that leads existing video file?

**Reply by kenany66** [reply]
I made a video explaining the problem that occurred to me. You can see it here in this link:
https://drive.google.com/file/d/1J2vrQZTittYHU8jEvv8zqh2xrW02UESp/view?usp=sharing

**Reply by SkalskiP** [reply]
@kenany66, and that's what I call a perfect bug report example. I know what the problem is. Your file names contain spaces. Linux machines don't really like that. There is some workaround, but it is always best practice to keep it simple and remove them. 

Most likely, you need to rename files at Google Drive to do that.

**Reply by kenany66** [reply]
I tried to rename the files but still the same problem.
https://drive.google.com/file/d/121acB23UkSAKDwHQBIemX4zOUCvIdNVg/view?usp=sharing
https://drive.google.com/file/d/1HURm6Vt30ovITTO5gzlL_lNQz7vvJwZH/view?usp=sharing
Therefore, I hope that you will make a tutorial for me on how to change the videos within your project. I will be very grateful to you.
